### PR TITLE
first try

### PR DESCRIPTION
**Pull request by CharkaouiMahdi**

### CODE REVIEW

File: src/lib/container.js [hunk 1]
```diff
@@ -7,35 +7,48 @@ let structureOptions = '';
...
-			if (componentName.charAt(i) === componentName.charAt(i).toUpperCase()) {
-				return true;
-			}
-		}
-		return false;
-	},
 
 	CaptalizeFirstLetter(componentName) {
 		if (componentName === componentName.toLowerCase()) {
 			console.log("1");
```
Comment (khofaai) [inline src/lib/container.js]: can you remove all `console` calls ?

File: src/lib/container.js [hunk 2]
```diff
@@ -7,35 +7,48 @@ let structureOptions = '';
...
-		} else if (componentName === componentName.toUpperCase()) {
+		}
+		else if (componentName === componentName.toUpperCase()) {
 			console.log("2");
-			let str = componentName.charAt(0);
-			for (let i = 1; i < componentName.length; i++) {
-				str = str + componentName.charAt(i).toLowerCase();
-			}
+			let str = componentName.charAt(0) + componentName.slice(1).toLowerCase();
+			// for (let i = 1; i < componentName.length; i++) {
```
Comment (khofaai) [inline src/lib/container.js]: please remove commented code :)

File: src/lib/container.js [hunk 3]
```diff
@@ -48,6 +61,7 @@ module.exports = {
 			structureTarget = target !== '' ? `/${target}` : '';
 			configurator.startCLI(componentName => {
 				componentName = this.CaptalizeFirstLetter(componentName);
+				componentName = this.EliminateSpaces(componentName);
```
Comment (khofaai) [inline src/lib/container.js]: why not use `EliminateSpaces` logic inside `CaptalizeFirstLetter` ?

File: src/lib/container.js (hunk 1 above)
Comment (CharkaouiMahdi) [reply src/lib/container.js]: yes sure

File: src/lib/container.js (hunk 3 above)
Comment (CharkaouiMahdi) [reply src/lib/container.js]: just thought it would be nice to separate them, no real purpose though

File: src/lib/container.js (hunk 2 above)
Comment (CharkaouiMahdi) [reply src/lib/container.js]: sure thing

**Pull request closed without merging**